lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -29,6 +29,7 @@
 #include "llvm/Support/Signals.h"
 #include "llvm/Support/Path.h"
 #include "llvm/Support/StringSaver.h"
+#include "llvm/Support/Error.h"
```

Comment:
`clang-format` will probably complain about the order here.

Reply:
Moved, thanks.